zulipterminal/model.py

```diff
@@ -184,6 +185,19 @@ def update_message(self, response: Dict[str, Any]) -> None:
                     self.msg_list.log[msg_pos] = new_msg_w
                     self.controller.loop.draw_screen()
 
+    def update_users_view(self, response: List[str]) -> None:
```

Comment:
You should update the `response` in self.users and get the whole user_list again.